File: Ki-Data/Ki-Data/src/main/java/com/app/Ki_Data/dto/CharacterPgDTO.java
package com.app.Ki_Data.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.List;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
public class CharacterPgDTO {
    private int id;
    private String name;
    private String race;
    private String gender;
    private int birthDate;
    private String description;
    private List<String> skillSet;
}

File: src/main/kotlin/com/app/Ki_Data/dto/CharacterPgDTO.kt
package com.app.Ki_Data.dto

data class CharacterPgDTO(
    val id: Int = 0,
    val name: String? = null,
    val race: String? = null,
    val gender: String? = null,
    val birthDate: Int = 0,
    val description: String? = null,
    val skillSet: MutableList<String>? = null
)